ozaria/engine/cinematic/dialogsystem/DialogSystem.js

```diff
@@ -8,9 +8,9 @@ import {
   getCamera
 } from '../../../../app/schemas/models/selectors/cinematic'
 import { processText, getDefaultTextPosition } from './helper'
+import { WIDTH, HEIGHT } from '../constants'
 
-const SVGNS = 'http://www.w3.org/2000/svg'
-const padding = 10
+const BUBBLE_PADDING = 10
```

Comment:
I like a separate constants file since there are a lot of cinematic constants, does it make sense to move `BUBBLE_PADDING` and `SPEECH_BUBBLE_MAX_WIDTH` there as well?

Reply:
I kept this here as it's domain specific to this system. Whilst the other constants are global cinematic constants. I think I prefer this, as it shows what constants are global to cinematics and which are local to a system. This is to remove cognitive overload when trying to understand how the Cinematics system works.